alpha/alpha-spring-cloud-starter-consul/src/main/java/org/apache/servicecomb/pack/alpha/server/discovery/consul/AlphaConsulAutoConfiguration.java

```diff
@@ -114,7 +114,11 @@ public void listenInstanceRegisteredEvent(InstanceRegisteredEvent instanceRegist
             NewService newservice =  new NewService();
             newservice.setName(service.getServiceName());
             newservice.setId(service.getServiceId());
-            List<String> tags = consulDiscoveryProperties.getTags();
+            newservice.setAddress(service.getAddress());
+            newservice.setPort(service.getServicePort());
+            newservice.setMeta(service.getServiceMeta());
```

Comment:
@fengbaicanhe  Thanks for the PR. When building with Spring-Boot 1.5.x, the CI got below error.

/home/travis/build/apache/servicecomb-pack/alpha/alpha-spring-cloud-starter-consul/src/main/java/org/apache/servicecomb/pack/alpha/server/discovery/consul/AlphaConsulAutoConfiguration.java:[119,39] cannot find symbol
  symbol:   method getServiceMeta()
  location: variable service of type com.ecwid.consul.v1.catalog.model.CatalogService

@coolbeevip  Could you take a look at this?

Reply:
Ok, I will look at this question later today.

Reply:
@fengbaicanhe 
I am not sure how to configure the consul meta parameter using spring cloud consul, Because I saw the following description on the https://cloud.spring.io/spring-cloud-consul/spring-cloud-consul.html#_metadata_and_consul_tags

> Consul does not yet support metadata on services. Spring Cloud’s ServiceInstance has a Map<String, String> metadata field. Spring Cloud Consul uses Consul tags to approximate metadata until Consul officially supports metadata. Tags with the form key=value will be split and used as a Map key and value respectively. Tags without the equal = sign, will be used as both the key and value.

Spring cloud consul uses the Consul Java API https://github.com/Ecwid/consul-api. Although the setMeta method is provided in the NewService class, spring cloud consul does not use this method, you can see it in the registration method of the ConsulAutoRegistration class of the spring-cloud-consul-discovery-2.1.1.RELEASE.jar

```java
  public static ConsulAutoRegistration registration(AutoServiceRegistrationProperties autoServiceRegistrationProperties, ConsulDiscoveryProperties properties, ApplicationContext context, List<ConsulRegistrationCustomizer> registrationCustomizers, List<ConsulManagementRegistrationCustomizer> managementRegistrationCustomizers, HeartbeatProperties heartbeatProperties) {
    NewService service = new NewService();
    String appName = getAppName(properties, context.getEnvironment());
    service.setId(getInstanceId(properties, context));
    if (!properties.isPreferAgentAddress()) {
      service.setAddress(properties.getHostname());
    }

    service.setName(normalizeForDns(appName));
    service.setTags(createTags(properties));
    if (properties.getPort() != null) {
      service.setPort(properties.getPort());
      setCheck(service, autoServiceRegistrationProperties, properties, context, heartbeatProperties);
    }

    ConsulAutoRegistration registration = new ConsulAutoRegistration(service, autoServiceRegistrationProperties, properties, context, heartbeatProperties, managementRegistrationCustomizers);
    customize(registrationCustomizers, registration);
    return registration;
  }
```
Do you have any suggestions about consul's meta?

Reply:
As you know, **Spring Cloud Consul uses Consul tags to approximate metadata until Consul officially supports metadata.** 
```
newservice.setMeta(service.getServiceMeta());
```
this line i committed is useless, it should be remove